web/src/containers/activity/EntryDetails.js

```diff
@@ -79,13 +79,13 @@ const EntryDetails = ({ activityIndex, fundingSource, activityKey, name }) => {
       >
         <Review heading={title} headingLevel={4} editContent={editContent} />
         <div className={collapsed ? 'visibility--print' : ''}>
-          <Overview activityIndex={activityIndex} />
-          <Goals aKey={activityKey} />
-          <Schedule aKey={activityKey} />
-          <Costs aKey={activityKey} />
-          <ContractorResources activityIndex={activityIndex} />
-          <CostAllocate aKey={activityKey} />
-          <StandardsAndConditions aKey={activityKey} />
+          <Overview activityIndex={index} />
+          <Goals aKey={activity.key} />
+          <Schedule aKey={activity.key} />
+          <Costs aKey={activity.key} />
+          <ContractorResources activityIndex={index} />
+          <CostAllocate activityIndex={index} aKey={activity.key} />
```

Comment:
This needs both the new `activityIndex` and the old `aKey` for now, because the key gets passed through to the `CostAllocateFFP` component (which is next on my list of components to fix).

Reply:
It looks like `CostAllocate` isn't using `aKey` anywhere - instead it's passing the activity key via `activity.key`, and it gets `activity` using `activityIndex`. Is that right? If so, I think you can get rid of `aKey` here.

Reply:
🤦‍♀ You're right! Done.